service/acm/api.go

```diff
@@ -107,6 +108,25 @@ func (c *ACM) AddTagsToCertificate(input *AddTagsToCertificateInput) (*AddTagsTo
 	return out, err
 }
 
+// AddTagsToCertificateEx is the same as AddTagsToCertificate with the addition of
```

Comment:
The method suffix here is incorrect; it should be `AddTagsToCertificateWithContext` rather than `AddTagsToCertificateEx`.